lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -181,14 +181,14 @@ def get(self):
             secret = self.server_config.get("openid-connect", "secret")
             client = self.server_config.get("openid-connect", "client")
             realm = self.server_config.get("openid-connect", "realm")
-            issuer = self.server_config.get("openid-connect", "server_url")
+            server = self.server_config.get("openid-connect", "server_url")
         except (NoOptionError, NoSectionError):
             pass
         else:
-            endpoints["openid-connect"] = {
+            endpoints["openid"] = {
```

Comment:
Perhaps "OpenID Connect" could be shortened to `oidc` if we really need it that short?

Reply:
Brevity is not the issue; what we're trying to do is to make it be a valid Javascript identifier, which means removing the hyphen.